rsopt/configuration/jobs.py

```diff
@@ -178,4 +183,4 @@ def setup(self, setup):
         if self._setup.setup.get('input_file'):
             self._setup.input_file_model = self._setup.parse_input_file(self._setup.setup.get('input_file'),
                                                                         self.setup.get('execution_type', False) == 'shifter',
-                                                                        self.ignored_files)
+                                                                        self.ignored_files())
```

Comment:
In setup.py, you're using this as a property. Do you want the same call mechanism here? Otherwise, it should probably renamed to `get_ignored_files()`

I'm actually confused as to why you need to call it here instead of having it be a property

Reply:
If `ignore_files` is a property it can't have an argument then, no?

Reply:
My fault -- you got rid of @property. Apologies.